includes/datatypes.php

```diff
@@ -661,6 +820,30 @@ function get_meta_fields() {
 			'type'    => 'select',
 			'options' => get_podcasting_categories_options(),
 		),
+		array(
+			'slug'  => 'podcasting_platforms',
+			'title' => __( 'Podcasting Platforms', 'simple-podcasting' ),
+			'type'  => 'platform_fields',
+		),
+		array(
+			'slug'    => 'podcasting_icon_theme',
```

Comment:
Nitpick:
I think keeping the icon theme option at block level is more user-friendly. Not a major issue, but love to hear your thoughts on this.

Reply:
@iamdharmesh that's a good idea, but something we can iteratively implement?

Reply:
Sure @Sidsector9, maybe we should create a follow-up issue for this?